aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -22,10 +33,17 @@ to be used across all HPE applications. For more information, see
 The Page Container contains the Page Content and the Page Header. 
 It is responsible for the layout for the main content of a page. 
 
---INSERT-EXAMPLE-HERE--
+<Example
+  caption='Diagram of a Page Container which contains a Page Header 
+    and the Page Content.'
+  pad="none"
+  previewWidth="min-content"
+>
+  <PageAnatomy />
+</Example>
 
 Jump to:
-- [Container Width](#container-width)
+- [Container Width](#page-container-widths)
```

Comment:
This section felt a bit disjointed from the content, left be wondering "why am I jumping there, I am not even sure what Grid is or Breakpoints?"